chia/cmds/wallet_funcs.py

```diff
@@ -68,6 +107,45 @@ async def get_transactions(args: dict, wallet_client: WalletRpcClient, fingerpri
                 break
 
 
+async def get_csv(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
+    wallet_id = args["id"]
+    offset = args["offset"]
+    file_path = args["file"]
+    date_fmt = args["date_fmt"]
+    delim = args["delim"]
+
+    txs: List[TransactionRecord] = await wallet_client.get_csv(wallet_id)
+    config = load_config(DEFAULT_ROOT_PATH, "config.yaml", SERVICE_NAME)
+    name = config["network_overrides"]["config"][config["selected_network"]]["address_prefix"]
+    if len(txs) == 0:
+        print("There are no transactions to this address")
+        return
+
+    header_list = [
+        "wallet_id",
+        "date_time",
+        "tx_type",
+        "status",
+        "height_confirmed",
+        "tx_name",
+        "amount",
+        "fee",
+        "to_address",
+    ]
+    file_header = delim.join(map(str, header_list))
+
+    lines = [get_csv_line(tx, name, date_fmt, delim) for tx in txs[offset:]]
+
+    if file_path:
+        f = open(file_path, "w")
+        f.write("\n".join([file_header] + lines))
```

Comment:
Did this look nice on file, have you tested it? 
If so, you can also give this textblock a name, since you assemble it below as well.